content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst

```diff
@@ -0,0 +1,80 @@
+Marketing activities
+====================
```

Comment:
This file needs a main heading, as right now the three headings "Marketing Activities", "Workflow activity" and "Child activities" are on the same level, and shown in the table of content.

![image](https://user-images.githubusercontent.com/19364378/205906660-acfeb403-f0c9-463c-9f33-aa94469eaee9.png)